data/korath/korath ships.txt

```diff
@@ -408,7 +408,7 @@ ship "Korath Raider Mk.II"
 	explode "large explosion" 40
 	explode "huge explosion" 10
 	"final explode" "final explosion large"
-	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, built and manufactured long before the era of shortages and desperation in life of exile."
+	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, built and manufactured before the era of shortages and desperation in life of exile."
```

Comment:
```suggestion
	description "The venerable Raider has long been a mainstay of the Korath raiding fleet, designed before the current era of shortages and desperation of life in exile."
```
- raiders may still be produced
- this change leaves open the possibility of it being designed either before or during exile
- grammar

Reply:
Done! Ambiguity for the win!